### DISCUSSION TITLE
【讨伐任务】「浊水粉碎幻灵」

### DISCUSSION THREAD

**Discussion by tingsha1935** [share121/inter-knot#2270] [其他]
![7426229667b939b5863eb4c0440833d](https://github.com/user-attachments/assets/a952a1db-2ae8-4901-b48d-963ba1fc1b86)
目击者情报

请各位把眼光放长远一些，[千织屋」将枫丹服装业推向了新的发展阶段，相应的，染料技术自然也需要大步跟进，才能应对越来越严格的色彩需求！作为向「千织屋』供货的染料制作商之一,我诚恳地请求有识之士的帮助——在海露站东边的小岛上，有两只蓝色系外表的魔物在徘徊，请击败它们，它们严重干扰了我们在附近收集新型染剂原料的工作！

目标情报
营地中，除了讨代目标「浊水粉碎幻灵」，还存在着与讨伐目标实力相当的另一只「浊水喷吐幻灵」与之一同行动。 
 
目标特点
根据调查，目标具有如下特性：
·风元素伤害无效
此外，目标具有以下弱点：
使用单手剑的角色造成的伤害提升
奖励：
【100枫丹声望】
【30000摩拉】

**Comment by Sqrootof** [comment]
牢伐，你也要没了

**Comment by LEARoTY** [comment]
这委托我接了！

**Comment by Momsaysnamesthataretoolongmakefoolslook** [comment]
这个委托我接了

**Comment by ALXY-12** [comment]
这个委托我接了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/01c96af07865b0d5215c9bc97ebeca73_8859049719826705182.png" alt="青雀 溜了" width="75" >

**Comment by Aha-SWYH** [comment]
我认识个人，是个道士，但是30000摩拉从璃月到枫丹的车费都不够吧？他可是很专业的，虽然有个有钱朋友但接委托总不能亏本吧
